bin/build-plugin-zip.sh

```diff
@@ -78,8 +78,8 @@ zip -r wc-admin.zip \
 	includes/**/*.php \
 	images/* \
 	$build_files \
-	languages/wc-admin.pot \
-	languages/wc-admin.php \
+	languages/woocommerce-admin.pot \
```

Comment:
Not sure if necessary or not, but should the build file also be renamed to `woocommerce-admin.zip`?

Reply:
Yeah I think that makes sense to change as well. 👍